sdks/java/io/jdbc/pom.xml

```diff
@@ -89,6 +89,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
The dependency was defined for the compile scope, but it was only used by the test classes so we can refine the scope to specific to test.